### ISSUE TITLE
Layout Designer: Can't save some forms

### ISSUE THREAD

**Issue by dasgarner** [xibosignage/xibo#1480]
Any form that doesn't contain a library select picker won't save properly.

**Comment by Vitoko** [comment]
This is also happening to me when I add a new event. Is it related? The solution did not work for me, not even in the design view, saving a twitter widget for example.

**Comment by dasgarner** [comment]
No, sorry, this issue only effects Widget forms on the Layout Designer - please post your issue on the community forum with logging/screenshots and we will try to help.
